Common/Securities/Cash.cs

```diff
@@ -43,18 +44,37 @@ public class Cash
         public event EventHandler Updated;
 
         /// <summary>
-        /// Gets the symbol of the security required to provide conversion rates.
+        /// Gets the symbol of the first security required to provide conversion rates.
         /// If this cash represents the account currency, then <see cref="QuantConnect.Symbol.Empty"/>
         /// is returned
         /// </summary>
-        public Symbol SecuritySymbol => ConversionRateSecurity?.Symbol ?? QuantConnect.Symbol.Empty;
+        [JsonIgnore]
+        [Obsolete("This property has been made obsolete, use SecuritySymbols instead.\n" +
+            "This property only returns the symbol of the first security used to calculate conversion rates, there may be more.")]
+        public Symbol SecuritySymbol =>
+            CurrencyConversion?.ConversionRateSecurities.First().Symbol ?? QuantConnect.Symbol.Empty;
 
         /// <summary>
-        /// Gets the security used to apply conversion rates.
+        /// Gets the first security used to apply conversion rates.
         /// If this cash represents the account currency, then null is returned.
         /// </summary>
         [JsonIgnore]
-        public Security ConversionRateSecurity { get; private set; }
+        [Obsolete("This property has been made obsolete, use CurrencyConversion.ConversionRateSecurities instead.\n" +
+            "This property only returns the first security used to calculate conversion rates, there may be more.")]
+        public Security ConversionRateSecurity => CurrencyConversion?.ConversionRateSecurities.First();
+
+        /// <summary>
+        /// Gets the symbols of the securities required to provide conversion rates.
+        /// If this cash represents the account currency, then an empty enumerable is returned.
+        /// </summary>
+        public IEnumerable<Symbol> SecuritySymbols =>
+            CurrencyConversion?.ConversionRateSecurities.Select(x => x.Symbol) ?? new List<Symbol>(0);
```

Comment:
nit - can replace `new List<Symbol>(0)` for static `Enumerable.Empty<Symbol>()`

Reply:
This appears to be incompatible with Newtonsoft.Json. Changing it to `Enumerable.Empty<Symbol>()` causes deserialization of a `Cash` instance in `LiveResult.CashBook` in [`LiveResult_NullChartPoint_IsSkipped`](https://github.com/QuantConnect/Lean/blob/master/Tests/Report/ResultDeserializationTests.cs#L99) to fail because Newtonsoft.Json tries to cast the empty `SecuritySymbols` enumerable to an `IList`:

![](https://user-images.githubusercontent.com/14951909/118876200-46d42400-b8ed-11eb-9131-fc9cbdacd3bd.png)

A similar bug was reported in Newtonsoft.Json in https://github.com/JamesNK/Newtonsoft.Json/issues/2034, but that one was fixed in Newtonsoft.Json 12.0.3, which is what all our projects appear to use. I cannot reproduce this error in an empty C# project, and debugging it inside LEAN doesn't get me anywhere, so I suggest keeping `new List<Symbol>(0)` for now as that does work.